static/src/js/components/AccessMethod/SwodlrForm.jsx

```diff
@@ -0,0 +1,439 @@
+import React, { useEffect, useState } from 'react'
+import PropTypes from 'prop-types'
+import {
+  Accordion,
```

Comment:
This isn't the best way to import from react-bootstrap, it should be 
```
import Accordion from `react-bootstrap/Accordion`
```
which is really annoying because of the auto import not working that way.

Also, sort the imports